File: jexler-war/src/main/webapp/WEB-INF/suite/filetailer.rb
import Java::net.jexler.StopSensor
import Java::net.jexler.FileTailerSensor

s = FileTailerSensor.new($jexler, "selftailer")
s.setFile($file.getAbsolutePath)
s.addFilterPattern("^import")
s.start
$sensors.add s

begin
  event = $events.take
  if event.is_a? FileTailerSensor::Event
    puts "Got line (ruby): " + event.getLine
  elsif event.is_a? StopSensor::Event
    return
  end
end while true

File: jexler-war/src/main/webapp/WEB-INF/suite/filetailer.py
from net.jexler import StopSensor
from net.jexler import FileTailerSensor

s = FileTailerSensor(jexler, "selftailer")
s.setFile(file.getAbsolutePath())
s.addFilterPattern("^from")
s.start()
sensors.add(s)

while True:
    event = events.take()
    if isinstance(event, FileTailerSensor.Event):
        print "Got line (jython): %s" % event.getLine()
    elif isinstance(event, StopSensor.Event):
        break
